OpenSim/Common/Storage.cpp

```diff
@@ -36,10 +36,27 @@
 #include "SimTKcommon.h"
 #include "GCVSpline.h"
 #include "StateVector.h"
+#include "STOFileAdapter.h"
+#include "TimeSeriesTable.h"
 
 using namespace OpenSim;
 using namespace std;
 
+void convertTableToStorage(const TimeSeriesTable& table, Storage& sto) {
```

Comment:
Making this private static would be fine with me.

We decided in a dev meeting to not support programmatically converting a TST to a Storage, to encourage migration to TST. So I didn't want to expose this function in the API.  I suppose I put this function only in the CPP file to further enforce that this is just a hidden utility.

As for why I pass in the storage rather than making this a member function: I think free functions (or equivalently, static member function) are easier to understand; you don't have to worry about how the function is affecting the state of the object so much. Also, I wanted to write a function that would be easy to move to the API if we decided to expose it (though perhaps making a static private function would be even easier to move into the API).

Reply:
Ok, thanks for the explanation. If the Dev Team doesn't want to support TST->Storage conversion, then I can see wanting to keep the method out of the header.